Makefile

```diff
@@ -0,0 +1,184 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.DEFAULT_GOAL:=help
+
+# variables
+TAG ?= dev
+ARCH ?= amd64
+IMAGE_PREFIX ?= ghcr.io/sovereigncloudstack
+BUILDER_IMAGE = $(IMAGE_PREFIX)/builder
+BUILDER_IMAGE_VERSION = $(shell cat .builder-image-version.txt)
+Version := $(shell git describe --tags --always --dirty)
+Commit := $(shell git rev-parse HEAD)
+
+# Certain aspects of the build are done in containers for consistency (e.g. protobuf generation)
+# If you have the correct tools installed and you want to speed up development you can run
+# make BUILD_IN_CONTAINER=false target
+# or you can override this with an environment variable
+BUILD_IN_CONTAINER ?= true
+
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+# Directories
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+BIN_DIR := bin
+TOOLS_DIR := hack/tools
+TOOLS_BIN_DIR := $(TOOLS_DIR)/$(BIN_DIR)
+export PATH := $(abspath $(TOOLS_BIN_DIR)):$(PATH)
+export GOBIN := $(abspath $(TOOLS_BIN_DIR))
+
+##@ Clean   
+#########
+# Clean #
+#########
+
+.PHONY: clean
+clean: ## cleans the csctl-plugin-openstack binary
+	@if [ -f csctl-plugin-openstack ]; then rm csctl-plugin-openstack; fi
+
+
+##@ Common
+##########
+# Common #
+##########
+.PHONY: build
+build: # build the csctl-plugin-openstack binary
+	go build -ldflags "$(LDFLAGS)" -o csctl-plugin-openstack main.go
```

Comment:
Do we need `"$(LDFLAGS)"` when it is not defined in this Makefile?

Reply:
updated via https://github.com/SovereignCloudStack/csctl-plugin-openstack/commit/c441a6cd91808346c4ac5d1b6e1eecfdd1db5529